src/components/CreditFilter.vue

```diff
@@ -333,7 +340,7 @@ export default defineComponent({
 
   &__mask {
     max-height: calc(3.4rem * 6.5 + 0.8rem); // タグ6個半 + padding
-    padding: $spacing-2 $spacing-0;
+    padding: $spacing-2 $spacing-2;
```

Comment:
> スクロールが発生する要素の左右のパディングの追加（スクロールバーで見づらくなってしまうので）
こちら助かりました! 1点、paddingはスクロールバーが表示される右サイドだけで良さそうです。

```suggestion
    padding: $spacing-2 $spacing-2 $spacing-2 0;
```